kubernetes/net-tests/net-test_ds.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: net-test-ds
+  labels:
+    app: net-test-ds
+spec:
+  selector:
+    matchLabels:
+      app: net-test
+  template:
+    metadata:
+      labels:
+        app: net-test
+    spec:
+      containers:
+      - name: iperf-server
+        image: kitesproject/net-test
```

Comment:
Where did you define this? You need to add instructions to the README on how to build the image with the right tag, and how to deploy the DaemonSet

Reply:
> How to build image with the right tag:
> 
> _docker build -t kitesproject/net-test:1.0 <pathWithDockerfile>_
> 
> To Deploy the DaemonSet, on the Master node, run:
> 
> _kubectl apply -f /vagrant/kubernetes/net-tests/net-test_ds.yaml_
> 
> And check with:
> 
> _kubectl get pod -o wide_
> 
> when the pods created are in the "Running" state

I write it in a separate README or in the main?

Reply:
main